docs/source/installation/troubleshooting.rst

```diff
@@ -182,3 +182,135 @@ operating system's documentation to find out where you can get it and how you
 have to install it.
 
 If you are unable to solve your problem, check out the `dvisvgm FAQ <https://dvisvgm.de/FAQ/>`_.
+
+(Windows) ``Python is not recognized as an internal or external command, operable program or batch file.``
+----------------------------------------------------------------------------------------------------------
+
+To fix this, you need to add the Python executable to your ``PATH`` environment variable.
+The following steps are due to `Michael Mrozek from a StackExchange answer <https://superuser.com/questions/143119/how-do-i-add-python-to-the-windows-path/143121#143121>`__.
```

Comment:
Actually, it would be best if you just linked the answer without copying it (after some discussion with @behackl).